### PR TITLE
Server name

### PR DESCRIPTION
**Pull request by noranda**
# Added server name

* Added a server name field on the rails side ([pull request]).
* Created a component called `server-details` that lists the server name and server URL.
* Added a button to add or edit a server name from the component itself.
* Added a delete button to the `servers` view to delete servers.
* Styled the component.

##### Styled Component
Component

##### Styled Component with Server Name Edit
Component with Edit

##### Styled component in unstyled view

Delete buttons added and implemented. Need to add verification.

Component on Servers

[pull request]:https://github.com/fhir-crucible/crucible/pull/11

### CODE REVIEW

File: app/controllers/test-runs/show.coffee [hunk 1]
```diff
@@ -23,6 +27,26 @@ TestRunsShowController = Ember.Controller.extend({
   actions:
     rerun: ->
       @transitionToRoute('servers.show', @get('model.server'))
+
```
Comment (jtferns) [inline app/controllers/test-runs/show.coffee]: Several lines in this controller appear in the ```server-details``` component...do we need to retain them here as well or can we remove the comments?

File: app/styles/_application.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+body {
```
Comment (jtferns) [inline app/styles/_application.scss]: Is this used to clear out styles on the ```body``` element or is it missing styles?

File: app/controllers/test-runs/show.coffee (hunk 1 above)
Comment (noranda) [inline app/controllers/test-runs/show.coffee]: Hey I totally forgot to remove those. Will do that now.

File: app/styles/_application.scss (hunk 2 above)
Comment (noranda) [inline app/styles/_application.scss]: It had a style, then I removed it, and now it's lonely. I will put it out of its misery.

**Pull request merged**